src/search/include/search/search.h

```diff
@@ -64,8 +63,8 @@ namespace details {
 template <typename Location, typename ActionType, typename ConstraintSymbolType>
 void
 label_graph(SearchTreeNode<Location, ActionType, ConstraintSymbolType> *node,
-            const std::set<ActionType> &                                controller_actions,
-            const std::set<ActionType> &                                environment_actions,
+            const std::set<ActionType>                                 &controller_actions,
```

Comment:
Short question on style here (I don't request any change): Why move the ampersand or the star to the variable name instead of making it part of the type? I know some styles do this, but I always wondered what the motivation behind this is.

Reply:
That was actually a bug in clang-format and newer versions of clang-format format in this way. I thought about reformatting everything but then figured it's better if we just reformat whenever we commit a file.

Reply:
Ah okay :). We can fix it in a clang-format config file, but tbh for me, it does not make much of a difference, so let's go with the proposed approach.